.github/workflows/rust.yml

```diff
@@ -268,26 +268,17 @@ jobs:
 
       - uses: actions-rs/toolchain@v1
         with:
-          toolchain: 1.48.0
+          toolchain: 1.56.1
```

Comment:
`rayon-core` requires 1.56.1, so that's our MSRV now.

Reply:
general query: is there a reason not to update the MSRV regularly anyway? It shouldn't matter for most things, right?

Reply:
> general query: is there a reason not to update the MSRV regularly anyway? It shouldn't matter for most things, right?

Initially I was trying to follow LTS releases or what Debian supported, thinking that it would make it easier to package there.
For example, current Debian stable (`bullseye`) is 1.48: https://tracker.debian.org/pkg/rustc
(Ubuntu releases it in the security channel, so even 18.04 has Rust 1.61: https://packages.ubuntu.com/bionic/rustc)

But... there are no sourmash packages in either of them. 

I don't think we should go updating MSRV every release, but 1.56.1 isa good one because it is the introduction of the 2021 edition.